tests/dtm/test_single_process_restart.py

```diff
@@ -150,24 +177,167 @@ def test_write_during_m0d_restart(self):
         proc_write_op.start()
 
         self.log.info("Step 2 : Perform Single m0d Process Restart During Write Operations")
-        self.dtm_obj.process_restart(self.master_node_list[0],
-                                      POD_NAME_PREFIX, MOTR_CONTAINER_PREFIX, self.m0d_process)
+        resp = self.dtm_obj.process_restart(master_node=self.master_node_list[0],
+                                            health_obj=self.health_obj,
+                                            pod_prefix=POD_NAME_PREFIX,
+                                            container_prefix=MOTR_CONTAINER_PREFIX,
+                                            process=self.m0d_process, check_proc_state=True)
+        assert_utils.assert_true(resp, "Failure in observed during process restart/recovery")
 
-        self.log.info("Step 3: Check hctl status if all services are online")
-        resp = self.health_obj.is_motr_online()
-        assert_utils.assert_true(resp, 'All services are not online.')
-
-        self.log.info("Step 4: Wait for Write Operation to complete.")
+        self.log.info("Step 3: Wait for Write Operation to complete.")
         if proc_write_op.is_alive():
             proc_write_op.join()
         resp = que.get()
         assert_utils.assert_true(resp[0], resp[1])
         workload_info = resp[1]
 
-        self.log.info("Step 5: Perform Read Operations on data written in Step 1:")
+        self.log.info("Step 4: Perform Read Operations on data written in Step 1:")
         self.dtm_obj.perform_ops(workload_info, que, True, True, True)
         resp = que.get()
         assert_utils.assert_true(resp[0], resp[1])
 
         self.test_completed = True
         self.log.info("ENDED: Verify READ during m0d restart using pkill")
+
+    # pylint: disable=too-many-statements
+    # pylint: disable-msg=too-many-locals
+    @pytest.mark.lc
+    @pytest.mark.dtm0
```

Comment:
why is there 0 in the dtm tag name?

Reply:
I just copied tags from existing tests. 
I will change it to dtm.